swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md

```diff
@@ -8,177 +8,184 @@ active: write-a-graphql-api-with-ballerina
 
-    resource function get isoCode() returns string {
-        return self.entryRecord.isoCode;
+    ```json
+    {
+        "iso_code": "AFG",
+        "country": "Afghanistan",
+        "cases": 159.303,
+        "deaths": 7.386,
+        "recovered": 146.084,
+        "active": 5.833
     }
+    ```
+    ![Create data record](/learn/images/featured-scenarios/write-a-graphql-api-with-ballerina/create-data-record.gif)
+    
+    The generated record will be as follows.
+
+    ```ballerina
+    type CovidEntry record {|
+        readonly string iso_code;
+        string country;
+        decimal cases?;
+        decimal deaths?;
+        decimal recovered?;
+        decimal active?;
+    |};
+    ```
+
+    In this code:
+    - The record type `CovidEntry` is defined.
+    - A single entry will include the ISO code for the location (`iso_code`), name of the country (`country`), number of cases (`cases`), number of deaths (`deaths`), number of recovered cases (`recovered`), and the number of active cases (`active`).
 
-    resource function get country() returns string {
-        return self.entryRecord.country;
-    }
+### Add data to the data source
 
-    resource function get cases() returns decimal? {
-        if self.entryRecord.cases is decimal {
-            return self.entryRecord.cases / 1000;
-        }
-        return;
+Follow the steps below to define the data [table](/learn/by-example/table).
+
+1. Create the table, as shown below.
+
+    >**Tip:** Enter `CovidEntry` as the table type, `iso_code` as the key, and `covidEntriesTable` as the variable type when creating the table.
+
+   ![Create data table](/learn/images/featured-scenarios/write-a-graphql-api-with-ballerina/create-data-table.gif)
+
+2. Replace the `{key: value}` placeholder of the generated table with the following entries to add the initial data to the table.
+
+    ```
+    {
+        iso_code: "AFG", 
+        country: "Afghanistan", 
+        cases: 159.303, 
+        deaths: 7.386, 
+        recovered: 146.084, 
+        active: 5.833
+    },
+    {
+        iso_code: "SL", 
+        country: "Sri Lanka", 
+        cases: 598.536, 
+        deaths: 15.243, 
+        recovered: 568.637, 
+        active: 14.656
+    },
+    {
+        iso_code: "US", 
+        country: "USA", 
+        cases: 69808.350, 
+        deaths: 880.976, 
+        recovered: 43892.277, 
+        active: 25035.097
     }
+    ```
 
-    resource function get deaths() returns decimal? {
-        if self.entryRecord.deaths is decimal {
-            return self.entryRecord.deaths / 1000;
-        }
-        return;
-    }
+The generated record and the table will be as follows.
```

Comment:
```suggestion
    The generated record and the table will be as follows.
```
check the indentation in following section.

Reply:
This is not indented as it belongs to both the 1st and 2nd bullet points.